raster/r.clump/clump.c

```diff
@@ -92,7 +94,9 @@ static CELL do_renumber(int *in_fd, DCELL *rng, int nin, int diag, int minsize,
             G_percent(row, nrows, 2);
 
             coffset = (off_t)row * csize;
-            lseek(cfd, coffset, SEEK_SET);
+            if (lseek(cfd, coffset, SEEK_SET) == -1) {
+                G_fatal_error(_("Seek error: %s"), strerror(errno));
```

Comment:
We need to come up with a better message here. "Seek error" sounds like "Seek help for an error" not "error of seek function" which is all what we are saying here.

"Unable to seek in a temp file: %s" - maybe not the best one possible but good at least in consistency with other error messages here and messages in `lib/segment/format.c`.

Reply:
done

Reply:
We have some similar (?) messages in ´lib/`:

```bash
ag seek | grep G_fatal
gis/seek.c:54:        G_fatal_error(_("Unable to seek: %s"), strerror(errno));
gis/seek.c:59:        G_fatal_error(_("Seek offset out of range"));
gis/seek.c:61:        G_fatal_error(_("Unable to seek: %s"), strerror(errno));
raster/get_row.c:911:        G_fatal_error(_("Error seeking null row %d for <%s>"), R, fcb->name);
segment/seek.c:43:        G_fatal_error("Segment seek: %s", strerror(errno));
segment/seek.c:52:        G_fatal_error("Segment seek: %s", strerror(errno));
```

It might be a good idea to streamline them (also good for the translations to homogenize them).

Reply:
Sure i will format them as well.

Reply:
Thanks (just to clarify: I meant to use the same wording for identical errors).

Reply:
May I suggest to change the message to `G_fatal_error(_("Unable to seek: %s"), strerror(errno));`, reusing the existing phrase from `G_fseek()` wrapper function. The `strerror()` message should make it clear what "seek" is about.

Reply:
@nilason so you are suggesting to keep the error message as "Unable to seek" in all files right.

Reply:
As this PR aims for a fix in r.clump, we should constrain the scope to just this file.

Reply:
ok then in that case i will modify the statements to this G_fatal_error(_("Unable to seek: %s"), strerror(errno));